### DISCUSSION TITLE
How to adjust the height of the Modal according to the screen?

### DISCUSSION THREAD

**Discussion by starxyn** [mantinedev/mantine#6600] [Q&A]
I have a open context modal which has the height cropped off when the screen gets smaller. How do I make it responsive?

**Comment by rtivital** [comment]
1. https://mantine.dev/core/modal/#usage-with-scrollarea
2. https://mantine.dev/core/modal/#fullscreen

**Reply by starxyn** [reply]
Instead of having scroll area, can I make the height responsive by being shorter to fit into the browser whenever the screen gets smaller? How do I do that?

**Reply by rtivital** [reply]
You can change component styles with Styles API. To change height depending on viewport size, use `vh` units.

**Reply by starxyn** [reply]
Does the styles api apply to modal manager? Can't find the styles api tab at modal manager

**Reply by rtivital** [reply]
Modals manager uses Modal component. See https://mantine.dev/styles/styles-api/#classnames-in-themecomponents
